qiskit_dynamics/solvers/solver_classes.py

```diff
@@ -88,53 +88,66 @@ class Solver:
 
     def __init__(
         self,
-        hamiltonian_operators: Array,
+        static_hamiltonian: Optional[Array] = None,
+        hamiltonian_operators: Optional[Array] = None,
         hamiltonian_signals: Optional[Union[List[Signal], SignalList]] = None,
+        static_dissipators: Optional[Array] = None,
         dissipator_operators: Optional[Array] = None,
         dissipator_signals: Optional[Union[List[Signal], SignalList]] = None,
-        drift: Optional[Array] = None,
         rotating_frame: Optional[Union[Array, RotatingFrame]] = None,
-        evaluation_mode: Optional[str] = "dense",
+        in_frame_basis: bool = False,
+        evaluation_mode: str = "dense",
         rwa_cutoff_freq: Optional[float] = None,
+        validate: bool = True,
     ):
         """Initialize solver with model information.
 
         Args:
+            static_hamiltonian: Constant Hamiltonian term. If a ``rotating_frame``
+                                is specified, the ``frame_operator`` will be subtracted from
+                                the static_hamiltonian.
             hamiltonian_operators: Hamiltonian operators.
             hamiltonian_signals: Coefficients for the Hamiltonian operators.
-            dissipator_operators: Optional dissipation operators.
+            static_dissipators: Constant dissipation operators.
+            dissipator_operators: Dissipation operators with time-dependent coefficients.
             dissipator_signals: Optional time-dependent coefficients for the dissipators. If
                                 ``None``, coefficients are assumed to be the constant ``1.``.
-            drift: Hamiltonian drift operator. If a ``rotating_frame`` is specified, the
-                   ``frame_operator`` will be subtracted from the drift.
             rotating_frame: Rotating frame to transform the model into.
+            in_frame_basis: Whether to represent the model in the basis in which the rotating
```

Comment:
Does this also mean its solved in the rotating frame basis? Might be good to make that clear in this docstring. Also should default be True instead of False? If you define a frame do you typically want to solve in that frame?

Reply:
See below - I can add somethign to the doc-string explaining the behaviour.